src/integrators/sdirk_integrators.jl

```diff
@@ -390,11 +407,8 @@ end
   end
 
   @. z = u - uprev
-  iter = 0
-  κ = cache.κ
-  tol = cache.tol
-
-  iter += 1
+  iter = 1
+  @. u = uprev + z
```

Comment:
By the way, the sam line is also present in the new SDIRK2 method: https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/integrators/sdirk_integrators.jl#L855